spec/recurly/billing_info_spec.rb

```diff
@@ -62,7 +62,7 @@
       )
       billing_info = BillingInfo.find 'sepa1234567890'
       billing_info.name_on_account.must_equal 'Account Name'
-      billing_info.iban.must_equal 'US1234567890'
```

Comment:
Given the removal of `iban`, it seems like test name should be updated. Maybe "must return an account's billing info as last two when available"?

Reply:
Good call out. I made some changes.